data/AgaKadela.json

```
@@ -0,0 +1,31 @@
+{
+  "name": "Aga Kadela",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "JavaScript Web Developer helping women become ✨ coding superheroes ✨ at Coder Woman",
+  "avatar": "https://avatars.githubusercontent.com/u/14938482?v=4",
```

```suggestion
  "avatar": "https://github.com/agakadela.png",
```
Please use this as your avatar

---

Done